src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -77,23 +129,87 @@ function vertical_diffusion_boundary_layer_tendency!(
 )
     ᶜρ = Y.c.ρ
     FT = Spaces.undertype(axes(ᶜρ))
-    (; ᶜp, ᶜspecific, sfc_conditions) = p.precomputed # assume ᶜts and ᶜp have been updated
-    (; C_E) = p.atmos.vert_diff
+    (; params) = p
+    thermo_params = CAP.thermodynamics_params(params)
+    (; ᶜts, ᶜp, ᶜspecific, sfc_conditions) = p.precomputed # assume ᶜts and ᶜp have been updated
+
+    κ = CAP.von_karman_const(params)
+    grav = CAP.grav(params)
+    
+    # TODO : Move to ClimaParameters
+    z₀ = FT(3.21e-5)
```

Comment:
The default z₀ for Monin Obukhov is 1e-5, see [here](https://github.com/CliMA/ClimaAtmos.jl/blob/765bbda92963d7e0b663a075a9241ca3aa63c21a/src/surface_conditions/surface_setups.jl#L22). If it's not too much work it would be good to make it a parameter. Otherwise it might be good to make these two the same.